webpack.config.js

```diff
@@ -233,6 +245,7 @@ module.exports = function (env = {}, argv = {}) {
     watchOptions: isCI ? { poll: 24 * 60 * 60 * 1000 } : {},
     devServer: {
       hot: false,
+      disableHostCheck: true,
```

Comment:
>disableHostCheck [](http://example.com/codeflow?start=6&length=16)

why this?

"THIS IS NOT RECOMMENDED as apps that do not check the host are vulnerable to DNS rebinding attacks."

Reply:
This is needed so that webpack hot reloading does not freak out when using <iframe> with sandbox. 

I noticed that somewhere in sockjs things went wrong when trying to read origin for the child <iframe> which is not possible for sandbox frames. Since we use hot reloading only in dev environment I only disabled host check to get around this issue there. This DOES NOT AFFECT PROD.